pyndemic/core/api.py

```diff
@@ -0,0 +1,59 @@
+from enum import Enum
+
+
+class CommandTypes(Enum):
+    EMPTY = 'empty'
+    CHECK = 'check'
+    COMMAND = 'command'
+    MESSAGE = 'message'
+    TERMINATION = 'termination'
+
+
+class ResponseTypes(Enum):
+    EMPTY = 'empty'
+    MESSAGE = 'message'
+    TERMINATION = 'termination'
+
+
+class GameplayCommands(Enum):
+    MOVE = 'move'
+    FLY = 'fly'
+    CHARTER = 'charter'
+    SHUTTLE = 'shuttle'
+    BUILD = 'build'
+    TREAT = 'treat'
+    CURE = 'cure'
+    SHARE = 'share'
+    PASS = 'pass'
+
+
+def termination_command():
+    command = {
+        'type': CommandTypes.TERMINATION.value,
```

Comment:
Let's use the Enum class objects as is, not their string representation. A serialization module can convert them when it's time.